easybuild/easyconfigs/l/LncLOOM/LncLOOM-2.0-environment.patch

```diff
@@ -0,0 +1,51 @@
+Binary files LncLOOMv2/.git/index and LncLOOMv2-edit/.git/index differ
```

Comment:
@zarybnicky This patch description doesn't make sense w.r.t. the changes being made?

It seems like it's basically two things:

* introducing environment variables for hardcoded paths
* tweaking the `biopython` version requirement

Can you reflect this better in the patch description, and mention yourself as an author of the patch?